File: test_cases/c/gpt/13.c
#include <assert.h>

int numDistinct(char* s, char* t);

int main(void) {
    assert(numDistinct("rabbbit", "rabbit") == 3);

    assert(numDistinct("babgbag", "bag") == 5);

    assert(numDistinct("abc", "abcd") == 0);

    assert(numDistinct("abcdef", "abcdef") == 1);

    char s[1001];
    for (int i = 0; i < 1000; i++) {
        s[i] = 'a';
    }
    s[1000] = '\0';
    assert(numDistinct(s, "a") == 1000);

    return 0;
}

File: test_cases/python/gpt/13.py
def run_tests(solution_module):
    f = solution_module.numDistinct
    assert f('rabbbit', 'rabbit') == 3
    assert f('babgbag', 'bag') == 5
    assert f('', 'a') == 0
    assert f('abc', '') == 1
    assert f('aaaaa', 'aa') == 10
